test-cases/features/tombstone_gc/longevity-tombstone-gc-modes.yaml

```diff
@@ -1,6 +1,6 @@
 test_duration: 200
 
-stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=10000 -clustering-row-size=200 -concurrency=10 -rows-per-request=1 -start-timestamp=SET_WRITE_TIMESTAMP -connection-count 10 -max-rate 30000 -timeout=120s -retry-number=30 -retry-interval=80ms,1s -duration=80m"]
+stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=10000 -clustering-row-size=200 -concurrency=10 -rows-per-request=1 -start-timestamp=SET_WRITE_TIMESTAMP -connection-count 10 -max-rate 30000 -timeout=120s -retry-number=30 -retry-interval=80ms,1s -duration=4m"]
```

Comment:
is 4m enough or just leftover from testing?
Btw. shouldn't it just insert all the intended data and end without overwrites?

Reply:
no, the test is modified due to former stability issues.
please read in title of last commit:
```
fix(test_switch_tombstone_gc_modes): run stress in cycles for each test step

Run stress per tombstone-gc-mode step of the test.
This way, stress writes are not mixed between different gc-modes and causes confusions.

```